core/src/filed/dir_cmd.cc

```diff
@@ -941,50 +931,6 @@ static bool job_cmd(JobControlRecord* jcr)
                     kBareosVersionStrings.GetOsInfo(), BAREOS_PLATFORM);
 }
 
-static bool RunbeforeCmd(JobControlRecord* jcr)
```

Comment:
Really interesting that these commands are not required anymore.  They are now using the `RunscriptCmd`, right?